mirrord-layer/src/error.rs

```diff
@@ -53,3 +56,40 @@ pub enum LayerError {
     #[error("mirrord-layer: Failed inserting listen, already exists!")]
     ListenAlreadyExists,
 }
+
+// mapping based on - https://man7.org/linux/man-pages/man3/errno.3.html
+
+macro_rules! impl_from {
+    ($type:ty, $return:expr) => {
```

Comment:
Just avoid using keywords here, I would go for `return_type` and `return_value`, instead of `type` and `return`.